2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -18,8 +18,8 @@ export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
 export MONGODB_QUIET=${MONGODB_QUIET:-true}
 
 # Constants used for waiting
-export MAX_ATTEMPTS=60
-export SLEEP_TIME=1
+MAX_ATTEMPTS=60
+SLEEP_TIME=1
```

Comment:
We may want to make these `readonly`, if they're really constants not intended to be tweaked.
Or if we want to make it possible to customize them, maybe something like `MAX_ATTEMPTS={MAX_ATTEMPTS:-60}`, WDYT?

Since we never tweaked these values (not possible with current implementation), it'd vote for `readonly`.